README.md

```diff
@@ -1,186 +1,94 @@
+our community and the projects we carry out for other organizations 
+that help them do good more easily or efficiently. They don't have much money 
+and software development is expensive. We simply try to bridge that gap 
+with our voluntary work to balance the scales.
 
-We use Docker for development purposes. To start working, your computer needs to support it.
+Second we develop the intranet for our community. We try to create the tools 
+which can help us organize ourselves better by storing the relevant information
+in one place, streamlining our practices and managing our responsibilities. 
+If you want to learn the general direction we try to accomplish, here you'll 
+find the general description of features we pursue at the moment:
+[Features we currently pursue](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues?q=is%3Aopen+is%3Aissue+label%3AEpic)
 
-### Commands
+### Wanna help us develop the product?
+* [Content - tasks that need some writing](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues?q=is%3Aopen+is%3Aissue+label%3A%22content+needed%22+sort%3Acreated-asc)
+* [Design - tasks that could use UX experise](https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues?q=is%3Aopen+is%3Aissue+label%3A%22UX+needed%22)
 
-| command      | what it does   |
-|--------------|----------------|
-| `make start` | starts project |
-| `make stop`  | stops project  |
-| `make logs`  | shows logs     |
-| `make test`  | runs tests     |
-| `make lint`  | runs linters   |
-
-Type `make help` to see other commands.
-
-### Default User password
-
-`pass123`
-
-### How to debug the app with remote database connection
+## How to get started
 
 ---
-> Please note that this step is optional.
-> You most likely do not need to follow it.
+> Join codeforpoznan_v3 channel on [our slack](https://join.slack.com/t/codeforpoznan/shared_invite/zt-8a7u52j8-yqB01C2YgYF4Lvd1pFM_jw) - this is the place where we discuss all the issues and help each other by sharing tips and knowledge. You can also ping [OtisRed](https://github.com/OtisRed) on github or @Otis in the slack DM if you need any help.
 ---
 
-Here's some information on how to set up the environment
-so that you will be connected to dev DB or produciton
-one using your local docker-compose setup.
-
-</br>
-
-#### 1. Copy `.envrc.example` to `.envrc`
-
-This is needed to avoid making changes to the
-`.envrc.example` file which could be commited by
-accident, because it's not ignored.  
-On the other hand, `.envrc` is ignored,
-so it's safe to modify it without ever worrying about
-commiting it by accident.
-
-
+#### 1. [Fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) onto your github account. 
 ```bash
-# run this in terminal, in the project directory
-cp .envrc.example .envrc
+# Open the terminal, choose the location on your computer where you want to keep it and paste in the command: 
+git clone git@github.com:CodeForPoznan/codeforpoznan.pl_v3.git
```

Comment:
Forked repository will result in different command, eg.: 
```git clone git@github.com:{name_of_your_user}/codeforpoznan.pl_v3.git```

Reply:
Good catch